src/init/features/hosting/github.ts

```diff
@@ -184,9 +186,27 @@ export async function initGitHub(setup: Setup, config: any, options: any): Promi
   logLabeledBullet("Action required", `Push any new workflow file(s) to your repo`);
 }
 
+function getGitFolderPath(): string {
+  const commandDir = process.cwd();
+  let projectRootDir = commandDir;
+
+  while (!fs.existsSync(path.resolve(projectRootDir, ".git"))) {
+    const parentDir = path.dirname(projectRootDir);
```

Comment:
I'm assuming this lobs off the current dirname `blah/blah` => `blah`

Reply:
yeah. I modeled this after https://github.com/firebase/firebase-tools/blob/24916de2be9740968798216bebb4daf30119d3da/src/detectProjectRoot.ts

Reply:
added a comment to hopefully make this clearer